Doc/Tutorial/chapter_advanced.tex

```diff
@@ -35,10 +35,9 @@ \section{Substitution Matrices}
 
 \subsection{SubsMat}
 
-This module provides a class and a few routines for generating substitution matrices, similar to BLOSUM or PAM matrices, but based on user-provided data. Additionally, you may select a matrix from MatrixInfo.py, a collection of established substitution matrices. The \verb+SeqMat+ class derives from a dictionary:
-\begin{minted}{python}
-class SeqMat(dict)
-\end{minted}
+This module provides a class and a few routines for generating substitution matrices, similar to BLOSUM or PAM matrices, but based on user-provided data. Additionally, you may select a matrix from MatrixInfo.py, a collection of established substitution matrices.
+
+The \verb+SeqMat+ class derives from a dictionary.
 The dictionary is of the form \verb|{(i1,j1):n1, (i1,j2):n2,...,(ik,jk):nk}| where i, j are alphabet letters, and n is a value.
```

Comment:
is the class part removed by black?

Reply:
No, not black (or tool for applying black to the documentation). If you look at the commit comments this was deliberate - it was hard to get a meaningful code snippet with valid syntax, and I didn't think it added much.